index.js

```diff
@@ -9,77 +9,43 @@ var __metadata = (this && this.__metadata) || function (k, v) {
     if (typeof Reflect === "object" && typeof Reflect.metadata === "function") return Reflect.metadata(k, v);
 };
 console.log('hello');
```

Comment:
```Index.js``` has to be added to ```.gitignore```, as it's a compiled file

Reply:
@Kregopaulgue thanks! I`ll fix it